libsolidity/parsing/Parser.cpp

```diff
@@ -831,6 +850,7 @@ ASTPointer<ModifierDefinition> Parser::parseModifierDefinition()
 
 	expectToken(Token::Modifier);
 	ASTPointer<ASTString> name(expectIdentifierToken());
+	SourceLocation nameLocation = currentLocation();
```

Comment:
```suggestion
	SourceLocation nameLocation = currentLocation();
	ASTPointer<ASTString> name(expectIdentifierToken());
```

Reply:
Maybe it would make sense to turn this into a helper?

Reply:
I agree. Done.